src/spatialdata/_core/query/spatial_query.py

```diff
@@ -585,9 +657,14 @@ def _(
         min_coordinate=min_coordinate,
         max_coordinate=max_coordinate,
     )
-    if bounding_box_mask.sum() == 0:
+    bounding_box_mask = np.where(bounding_box_mask.compute())[0]
```

Comment:
nitpick, but these are indices, right? I think a mask is usually a bool.

Reply:
true, I changed this